pkg/reconciler/delivery/delivery_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package delivery
+
+import (
+	"testing"
+
+	v1 "knative.dev/serving/pkg/apis/serving/v1"
+	. "knative.dev/serving/pkg/testing/v1"
+)
+
+func TestShouldSkipConfig(t *testing.T) {
+	var tests = []struct {
+		name  string
+		cfg  *v1.Configuration
+		want  bool
+	}{
+		{"Namespace_match_but_not_name", Configuration(KCDNamespace, "random"), false},
+		{"Name_match_but_not_namespace", Configuration("random", KCDName), false},
+		{"Both_match", Configuration(KCDNamespace, KCDName), true},
+		{"None_match", Configuration("random_namespace", "random_name"), false},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ans := shouldSkipConfig(tt.cfg)
+			if ans != tt.want {
```

Comment:
use `cmp.Diff`

example: https://github.com/googleinterns/knative-continuous-delivery/blob/b01e6028adfcdfdf667d9357430f65d8e785c878/vendor/knative.dev/pkg/reconciler/testing/table.go#L178

Reply:
I will use this for comparing more complex objects in the future